playbooks/roles/openconnect/templates/instructions.md.j2

```diff
@@ -1,18 +1,32 @@
 OpenConnect / Cisco AnyConnect
 ------------------------------
 
-OpenConnect is an extremely high-performance and lightweight VPN server that also features full compatibility with the official Cisco AnyConnect clients. The [protocol](http://www.infradead.org/ocserv/technical.html) is built on top of standards like HTTP, TLS, and DTLS, and it's one of the most popular and widely used VPN technologies among large multi-national corporations. This means that in addition to its ease-of-use and speed, OpenConnect is also highly resistant to censorship and is almost never blocked.
+OpenConnect is an extremely high-performance and lightweight VPN server that also features full compatibility with the official Cisco AnyConnect clients. The [protocol](http://www.infradead.org/ocserv/technical.html) is built on top of standards like HTTP, TLS, and DTLS, and it's one of the most popular and widely used VPN technologies. Due to its use among large multi-national corporations, it often means that at the protocol level, it is seldom targetted for censorship.
 
 ---
 * Platforms
+  * [Certificates](#clientcerts)
   * [Windows](#windows)
   * [OS X](#osx)
   * [Linux](#linux)
   * [Android](#android)
   * [iOS](#ios)
 
+<a name="clientcerts"></a>
+### Server and client Certifcates (OS X, Android, iOS) ###
+
+Certificates are a mechanism by which through [public key crypytography](https://en.wikipedia.org/wiki/Public-key_cryptography) allow for secure authentication to a given resource. OS X, Andoid and iOS OpenConnect/AnyConnect support this form of authentication, and is the recommended approach to connect to your Streisand OpenConnect VPN.
```

Comment:
I think this initial sentence could be simplified somewhat. I would probably skip the link to the wiki on asymmetric encryption all together and stick with:
> Client certificates are a mechanism by which clients can authenticate themselves securely with the server.